api/services/S3SiteRemover.js

```diff
@@ -10,25 +11,24 @@ const s3Client = new AWS.S3({
 
 /**
   Deletes the array of S3 objects passed to it.
-
   Since AWS limits the number of objects that can be deleted at a time, this
   method deletes the objects 1000 at a time. It does so recursively so each
   group of 1000 is deleted one after the other instead of simultaneously. This
   prevents the delete requests from breaking AWS's rate limit.
 */
-const deleteObjects = (objects) => {
-  if (objects.length === 0) {
+const deleteObjects = (keys) => {
+  if (keys.length === 0) {
     return Promise.resolve();
   }
 
-  const objectsToDeleteNow = objects.slice(0, 1000);
-  const objectsToDeleteLater = objects.slice(1000, objects.length);
+  const keysToDeleteNow = keys.slice(0, 1000);
```

Comment:
Do we have a global constants file for the backend? Might be nice to use the S3_MAX_FILES constant here, so there is some context for why we choose 1000

Reply:
good idea. the changes in this file were mostly just renaming variables, so i didn't think too much about them.